README.md

```diff
@@ -25,7 +25,9 @@ To be able to run your benchmarks, current-bench assumes certain things about yo
 - You have a `.opam` file at the root of your project specifying your dependencies and all the usual metadata.
 - Your benchmarks can be run by using `make bench` at the root of your project. If you don't have one, you'll need a `Makefile`. (For example, see [this simple Makefile](https://github.com/example-ocaml-org/my-ocaml-project/blob/main/Makefile))
 - Either you want OCaml 4.14, or you have a custom `bench.Dockerfile` at the root of your project that installs the necessary system dependencies (including opam) and the correct OCaml version.
-- The results of the benchmarks are in json, with a few specific fields, see below for the exact format.
+- The results of the benchmarks are in json, with a few specific fields, see below for the exact format. The standard output of `make bench` will be searched for json matching this format.
+- If a `bench.Dockerfile` is present then benchmarks will be run from within a docker container built from this file. This means `bench.Dockerfile` must set up an image containing a `Makefile` which runs benchmarks by running `make bench`.
+- Benchmarks only run when a pull request is opened and when pushing to a remote branch which is the source of a pull request.
```

Comment:
Benchmarks are also run on the main branch, on new pushes, right?

Reply:
Not in my experience, although I might have just missed them in the UI or something. Just in case I've missed it, can you point me to where I can find the main branch benchmarks for this project: https://autumn.ocamllabs.io/gridbugs/dune-monorepo-bench

Reply:
oh wait I think I found it. Do you just click on the branch name at the top of the screen? I think I might have just never pushed a working benchmark to the main branch of my repo before. I'll try it out now.

Reply:
ok I just pushed to the main branch and I don't see any activity on autumn. Let me know if I'm missing something!

Reply:
(For some reason, the last `main` benchmark failed on `b49df1` from 3 days ago and the following commits were not seen... I'm not sure that we receive any webhooks from github when `main` is updated, so I guess we only check for updates to main when a PR is opened/updated/closed/merged (?))

Reply:
I updated the point about when benchmarks run to include merging pull requests.